docs/contributors/contributors-guide.md

```diff
@@ -0,0 +1,178 @@
+---
+id: contributors-guide-to-open-sauced
+title: 'Contributors Guide to OpenSauced'
+sidebar_label: Contributors Guide to OpenSauced
+keywords:
+  - contributors
+  - guides
+---
+
+## Introduction to OpenSauced for Contributors
+
+OpenSauced is a platform dedicated to empowering you, the contributor, as you grow in your open source journey. We offer a GitHub-powered dashboard for tracking your contributions and gaining insights into your favorite open source projects. You can also customize your Insights Pages and Lists with the repositories you choose and the contributors you want to see.
+
+## Getting Started
+
+To get started, you will need to create an account on OpenSauced. You can do this by visiting the [app.opensauced.pizza](https://app.opensauced.pizza) and clicking the "Connect with GitHub" button. You will be prompted to sign in with your GitHub account. Once you have signed in, you will be redirected to the OpenSauced dashboard.
+
+## Insights: Connecting Your Repositories
+<!-- revise this one tomorrow -->
+OpenSauced Insights provides contributors with a visual representation of your favorite project's health and contributor engagement. This feature also offers real-time data and analytics to understand contribution patterns, identify key contributors, and measure the project's growth. Insights can help you assess team activity, understand when bottlenecks occur, identify where resource allocation is needed, plan and identify key contributors, understand alumni contributors, or spot emerging trends.
+
+Insights features allow you to:
+
+- track the contributions of a specific team, a specific repository, or a group of repositories,
+- view the number of pull requests that are open, closed, in draft, and merged,
+- view the average number of days it takes for a pull request to be merged (PR velocity),
+- identify the top contributors to your project,
+- identify the current activity level and trends in activity of your contributors.
+
+### Creating a New Insight Page
+<!-- work on this tomorrow -->
+To connect your repositories, click the "Insights" tab on the dashboard. You will be redirected to a page where you can create your new Insight Page.
+
+There are two ways to add repositories to your Insight Page:
+
+1. **Sync GitHub organization**. Syncing your GitHub organization is a good idea for someone who seeks a comprehensive, unified view of your organization's open source project activities and trends, enabling streamlined management, strategic decision-making, and efficient allocation of resources across multiple repositories.
+2. **Connect individual repositories**. Connecting individual repositories is a good idea for someone who wants to track the contributions of a specific repository or group of repositories. For example, you might want to track projects you maintain that are not under the same organization, or you may want to track projects that align with your team's objectives.
```

Comment:
We want to remove any mention of "projects you maintain" bc this guide is for contributors. So for instance, we have this currently:

> **Connect individual repositories**. Connecting individual repositories is a good idea for someone who wants to track the contributions of a specific repository or group of repositories. For example, you might want to track projects you maintain that are not under the same organization, or you may want to track projects that align with your team's objectives.

Let's update to this:

> **Connect individual repositories**. Connecting individual repositories is a good idea for someone who wants to track the contributions of a specific repository or group of repositories. For example, you might want to track projects you have contributed to or would like to contribute to in the future.